api/src/main/resources/applicationContext-service.xml

```diff
@@ -582,20 +582,8 @@
 	<!-- interceptors are used in order of name, this needs to be one of the first and should be since auditable happens to start with 'a' -->
 	<bean id="auditableInterceptor" class="org.openmrs.api.db.hibernate.AuditableInterceptor" />
 	
-	<bean id="sessionFactory" class="org.openmrs.api.db.hibernate.HibernateSessionFactoryBean">
-		<property name="configLocations">
-			<list>
-				<value>classpath:hibernate.cfg.xml</value>
-			</list>
-		</property>
-		<property name="mappingJarLocations">
-			<ref bean="mappingJarResources" />
-		</property>
-        <property name="packagesToScan">
-            <list>
-                <value>org.openmrs</value>
-            </list>
-            </property>
+	<bean id="sessionFactory" class="org.springframework.orm.jpa.vendor.HibernateJpaSessionFactoryBean">
```

Comment:
I had to edit this to make it work together with JPA according to http://stackoverflow.com/questions/21671731/simultaneous-use-of-hibernate-and-spring-data-jpa